QgisModelBaker/gui/panel/export_models_panel.py

```diff
@@ -0,0 +1,76 @@
+"""
+/***************************************************************************
+        begin                : 22.11.2021
+        git sha              : :%H$
+        copyright            : (C) 2021 by Dave Signer
+        email                : david at opengis ch
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+"""
+
+from qgis.PyQt.QtWidgets import QWidget
+
+import QgisModelBaker.utils.gui_utils as gui_utils
+from QgisModelBaker.utils import gui_utils
+
+WIDGET_UI = gui_utils.get_ui_class("export_models_panel.ui")
+
+
+# could be renamed since it's not only model - it's dataset and basket as well
+class ExportModelsPanel(QWidget, WIDGET_UI):
+    def __init__(self, parent=None):
+        QWidget.__init__(self, parent)
+        self.setupUi(self)
+        self.parent = parent
+
+    def setup_dialog(self, validation=False):
+        self._generate_texts(validation)
+        self.export_models_checkbox.setChecked(False)
+        self.items_view.setVisible(False)
+        if self.parent:
+            self.items_view.setModel(self.parent.current_export_models_model)
+            self.items_view.clicked.connect(self.items_view.model().check)
+            self.items_view.space_pressed.connect(self.items_view.model().check)
+
+            self.export_models_checkbox.stateChanged.connect(self._active_state_changed)
+            self.export_models_checkbox.setChecked(
+                self.parent.current_export_models_active
+            )
+            self._active_state_changed(self.parent.current_export_models_active)
```

Comment:
When `self.parent.current_export_models_active` is `True`, `_active_state_changed` will be called twice. Line 43 and 44 should be swapped